packages/compass-import-export/src/utils/csv.ts

```diff
@@ -293,11 +293,15 @@ export function parseValue(
     if (ISO_DATE.test(value)) {
       // iso string
       date = new Date(value);
-    } else {
-      // fall back to assuming it is an int64 value
+    } else if (!isNaN(+value)) {
```

Comment:
Bug found with e2e tests: I made the assumption that anything that doesn't match the regex can only be an int or not a date. But the old code just passed whatever we got to `new Date()`, so things like YYYY-MM-DD parsed. Luckily we had some test fixtures that happened to hit that.